product-service/dal/getDBClient.ts

```diff
@@ -0,0 +1,14 @@
+import {Client} from 'pg';
+
+export const getDBClient = () => {
+    const { PG_HOST, PG_PORT, PG_DATABASE, PG_USERNAME, PG_PASSWORD } = process.env;
+    const dbOptions = {
```

Comment:
It is worth to keep dbOptions in separate file because you use the same for Pool

Reply:
dbOptions moved to separate file for DB clients.